lib/main_containers.pm

```diff
@@ -243,9 +243,6 @@ sub load_container_tests {
                 load_image_tests_in_k8s($run_args) if (/k8s/i);
                 load_image_tests_in_openshift if (/openshift/i);
             }
-        } elsif (get_var('REPO_BCI')) {
-            loadtest 'containers/host_configuration';
-            loadtest 'containers/bci_repo';
```

Comment:
wait, if you remove this, how is these jobs now scheduled?

Reply:
It would be scheduled by ibs sync and I have put the related variables into job group.

Reply:
Ah ok, it's scheduled because is_container_image_test() returns 1 due to `BCI_TESTS=1`.